lib/LanguageContext.ts

```diff
@@ -1,4 +1,4 @@
 import { createContext } from 'react';
-import { Lang } from './types';
+import type { Lang } from './types';
```

Comment:
a bunch of the random churn in the diffs is because i have my neovim set to auto fix lint stuff with the same rules as our gertie web repo. let me know if that bothers you.

Reply:
No that's great. Is there a way to set that up in vscode?